src/app/components/mayastor-dialog/mayastor-dialog.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, OnInit } from '@angular/core';
+import { DashboardData } from 'src/app/services/ci-dashboard.service';
+import { ActivatedRoute, Router } from '@angular/router';
+
+
+@Component({
+  selector: 'app-mayastor-dialog',
+  templateUrl: './mayastor-dialog.component.html',
+  styleUrls: ['./mayastor-dialog.component.scss']
+})
+export class MayastorDialogComponent implements OnInit {
+  public issueID: Number;
+  public testData: any;
+  constructor(private activatedRoute: ActivatedRoute, private ApiService: DashboardData) { }
+  ngOnInit(): void {
+    this.activatedRoute.params.subscribe((params) => {
+      this.issueID = params.id;
+      console.log(params.id);
+    });
+    this.ApiService.getMayastorTest().subscribe((res) => {
+      const tests = res.filter((test) => test.issueId === this.issueID);
+      
+      this.testData = tests[0];
+      console.log(this.testData);
+
+    });
+  }
+
+  statusIcon(status) {
+    switch (status) {
+      case 'PASSED':
+        return 'far fa-check-circle text-success mx-2 font-size-18';
+      case 'FAILED':
+        return 'far fa-times-circle text-danger mx-2 font-size-18';
+      case 'running':
```

Comment:
why the above are capital case and these are lower case. Lets bring these from enum.